CHANGELOG.md

```diff
@@ -53,6 +53,7 @@ For older changes see the [archived Singularity change log](https://github.com/a
 ### Bug fixes
 
 - Add specific error for unreadable image / overlay file.
+- Ensure cgroups device limits are default allow per past behavior.
```

Comment:
Actually we don't need this CHANGELOG entry because we haven't released the code it is fixing, so please remove that.